test/unittest/CMakeLists.txt

```diff
@@ -37,9 +37,6 @@ target_compile_options(
 )
 
 target_link_libraries(unittests Catch2::Catch2)
-if(ENABLE_TCMALLOC)
```

Comment:
Same, we we are removing it here?

Reply:
create_test_executable is setting the option so we don't need it twice.